darwin/exporter/formats/nifti.py

```diff
@@ -510,6 +508,27 @@ def unnest_dict_to_list(d: Dict) -> List:
         nib.save(img=img, filename=output_path)
 
 
+def get_reoriented_nifti_image(img: nib.Nifti1Image, volume: Dict) -> nib.Nifti1Image:
+    """
+    Reorients the given NIfTI image based on the original affine.
+
+    Parameters
+    ----------
+    img: nib.Nifti1Image
+        The NIfTI image to be reoriented
+    volume: Dict
+        The volume containing the affine and original affine
+    """
+    if volume.original_affine is not None:
+        img_ax_codes = nib.orientations.aff2axcodes(volume.affine)
```

Comment:
I don't think this is correct for pre MED_2D -- the volume.affine is (expected to be at least) RAS

So this will try to go from RAS -> original orientation.

But really its meant to go from LPI -> original orientation. I think this is why `dataobj=np.flip(volume.pixel_array, (0, 1, 2)).astype(np.int16)` used to be invoked: to go from LPI to RAS so that this logic would make sense for pre MED_2D files

Reply:
Updated